scripts/hints.sh

```diff
@@ -37,6 +37,13 @@ function show_hints_and_swap() {
   current_pane_id=$1
   fingers_pane_id=$2
   compact_state=$3
-  show_hints "$fingers_pane_id" $compact_state
-  tmux swap-pane -s "$current_pane_id" -t "$fingers_pane_id"
+  pane_was_zoomed=$4
+
+  if [[ $pane_was_zoomed == 1 ]]; then
+    show_hints "$fingers_pane_id" $compact_state
+    tmux swap-pane -s "$current_pane_id" -t "$fingers_pane_id"
+  else
+    tmux swap-pane -s "$current_pane_id" -t "$fingers_pane_id"
+    show_hints "$fingers_pane_id" $compact_state
```

Comment:
Hi @vfreefly, thanks for your contribution! 🙏

I've tested it locally and I've realized that there is now some flickering.  That's because `show_hints` needs to be called before swapping the pane, otherwise there is a split of a second where you can see hints rendering.

I think the underlying issue is that the `fingers_pane` and `current_pane` are different size, and when they are swapped there is some text-wrapping going on that messes things up.

One way to solve it would be to call 'force-width' and 'force-height' ( more info in `man tmux` ) when [initializing fingers pane](https://github.com/morantron/tmux-fingers/blob/9089bf26a18d2d79ee7504de099d5fc4c80ac6f0/scripts/tmux-fingers.sh#L6).  Current pane width and height should be in environment variables `$COLUMNS` and `$LINES`.

Let me know if you want to give it a try :raised_hands:

Thanks